From e382083ab81071738bfe80d2e3be5c51cec5fd61 Mon Sep 17 00:00:00 2001
From: Dragos Tatulea <dragos@endocode.com>
Date: Wed, 9 Nov 2016 18:43:15 +0100
Subject: [PATCH 32/58] src: add LTE atom define

---
 src/ofono.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ofono.h b/src/ofono.h
index 5b37a64..1df47d5 100644
--- a/src/ofono.h
+++ b/src/ofono.h
@@ -153,6 +153,7 @@ enum ofono_atom_type {
 	OFONO_ATOM_TYPE_HANDSFREE,
 	OFONO_ATOM_TYPE_SIRI,
 	OFONO_ATOM_TYPE_NETMON,
+	OFONO_ATOM_TYPE_LTE,
 };
 
 enum ofono_atom_watch_condition {
-- 
2.7.4

